src/_base/harness/config/pipeline.yml

```diff
@@ -1,9 +1,10 @@
 command('app publish'): |
   #!bash(workspace:/)|@
-  run docker login -u="@('docker.username')" -p="@('docker.password')" @('docker.repository')
```

Comment:
By removing the docker.repository, we can't log into for example `quay.io` or `gcr.io`, only docker hub.

Reply:
ok, so I should use repository as `docker.io/<repourl>` which also allows login to dockerhub.

Reply:
No just need to put this back to how it was - set `docker.repository` as the whole domain/repo name, e.g. "hello/example" for docker hub (or "docker.io/hello/example"), or "quay.io/hello/example", etc.

Reply:
"hello/example" works in docker pull but does not allow to login, only "docker.io/hello/example" works in both cases

Reply:
True, perhaps we should split the domain from the repo name?